js/app/controllers/content-product.ng

```diff
@@ -77,6 +77,124 @@ Set-GceInstance "instance-name" -RemoveMetadata "newKey"
 Set-GceInstance "instance-name" -RemoveTag "beta" -AddTag "alpha"</pre>  
   </div>
 
+    <!-- Container Engine -->
+  <div ng-switch-when="google-cloud-container">
+    <p>
+      Google Container Engine is a powerful cluster manager and orchestration
+      system for running your Docker containers. Container Engine schedules
+      your containers into the cluster and manages them automatically based
```

Comment:
"Container Engine schedules your containers into the cluster"

I'm not familiar yet with the terminology for Containers, but this sentence sounds a little awkward. If this phrase is the proper terminology, then please disregard.